lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -73,19 +84,51 @@ def before_scenario_starts(scenario)
       end
 
       def before_execution_starts(execution)
-        @spinner.update(execution_info(execution, 'running'))
-        @spinner.activate
+        puts(execution_info(execution, ''))
+      end
+
+      def print(string)
+        new_line_if_needed
+        @stdout.print(string)
+        @stdout.flush
+        record_last_line(string)
       end
 
-      def on_execution_update(execution, update)
```

Comment:
I was also going to ask this, what's different between `puts` and `print`? (In `foreman_maintain` of course, and not ruby :smile:)